samples/cpp/benchmark_app/main.cpp

```diff
@@ -606,11 +606,14 @@ int main(int argc, char* argv[]) {
                 device_nstreams.erase(device);
             }
         }
-
-        for (auto&& item : config) {
-            core.set_property(item.first, item.second);
-        }
-
+        auto result = std::find_if(config.begin(), config.end(), [&](const std::pair<std::string, ov::AnyMap>& item) {
+            if (device_name.find(item.first) == 0)
```

Comment:
Why remove core.set_property() here completely?

Reply:
Would it work in case when device_name is just name of meta plugin and device priorities are specified in config file, for example? Or in case of AUTO plugin?

Reply:
> Why remove core.set_property() here completely?

we disabled core.set_property() to support ov::device::properties as explained from wangle's [comment](https://github.com/openvinotoolkit/openvino/pull/15175#discussion_r1091852013). Need to update benchmark app to avoid setting ov::device::properties through this core.set_property(). just move all of the properties setting into the step of  compile_model() that supports all kind of properties setting.

Reply:
> > Would it work in case when device_name is just name of meta plugin and device priorities are specified in config file, for example? Or in case of AUTO plugin?
> 
> You mean device name is meta plugin and configurate device priorities for meta plugin in the config file? if yes, it will work. For example, -d AUTO and configuration in config file looks like this. `{ "AUTO": { "PERFORMANCE_HINT": "", "PERF_COUNT": "NO", "MULTI_DEVICE_PRIORITIES" : "GPU"} }`

I mean if I set -nstreams GPU:2 in command line for such configuration. GPU is absent in AUTO name

Reply:
> 

Yes, it work. nstreams for GPU will be set as secondary property to AUTO. Then

- If AUTO select GPU as target device, nstreams for GPU will be set to GPU plugin as well. 
- If AUTO select CPU as target device, nstreams for GPU will be ignored.